Prefabs/CameraRig/SimulatedCameraRig/SharedResources/Scripts/SimulatorConfigurator.cs

```diff
@@ -0,0 +1,103 @@
+﻿namespace VRTK.Prefabs.CameraRig.SimulatedCameraRig
+{
+    using UnityEngine;
+    using UnityEngine.XR;
+    using Malimbe.XmlDocumentationAttribute;
+    using Malimbe.PropertySerializationAttribute;
+    using Zinnia.Extension;
+    using Zinnia.Data.Attribute;
+
+    /// <summary>
+    /// Sets up the Simulated CameraRig Prefab based on the provided user settings.
+    /// </summary>
+    public class SimulatorConfigurator : MonoBehaviour
+    {
+        #region Facade Settings
+        /// <summary>
+        /// The public facade.
+        /// </summary>
+        [Serialized]
+        [field: Header("Facade Settings"), DocumentedByXml, Restricted]
+        public SimulatorFacade Facade { get; protected set; }
+        #endregion
+
+        #region Control Settings
+        /// <summary>
+        /// The linked ActiveObjectController for the PlayArea.
+        /// </summary>
+        [Serialized]
+        [field: Header("Control Settings"), DocumentedByXml, Restricted]
+        public ActiveObjectController PersonObjectController { get; protected set; }
+        /// <summary>
+        /// The linked ActiveObjectController for the Left Controller.
+        /// </summary>
+        [Serialized]
+        [field: DocumentedByXml, Restricted]
+        public ActiveObjectController LeftObjectController { get; protected set; }
+        /// <summary>
+        /// The linked ActiveObjectController for the Right Controller.
+        /// </summary>
+        [Serialized]
+        [field: DocumentedByXml, Restricted]
+        public ActiveObjectController RightObjectController { get; protected set; }
+        #endregion
+
+        /// <summary>
+        /// The original configuration of XRSettings.
+        /// </summary>
+        protected bool originalXRSettings;
```

Comment:
```suggestion
        protected bool wereXRSettingsEnabled;
```
A suggestion to honor the .NET naming conventions around booleans (should use `is`/`was`).